lib/nice_partials/monkey_patch.rb

```diff
@@ -79,6 +90,9 @@ def capture_with_outer_partial_access(*arguments, &block)
 module NicePartials::PartialRendering
   ActionView::PartialRenderer.prepend self
 
+  # Automatically captures the `block` in case the partial has no manual capturing `yield` call.
+  #
+  # Effectively the same as having `<% yield partial %>` as the first partial line.
```

Comment:
Nit pick: this is fine, although "effectively the same" adds a little wiggle room in there. Do you think it would be even more helpful to state "this is the same as X" and if it's not exactly the same, describe the differences of two things that are very similar, but have a subtle difference? You might have to break out another docs section though. I guess it depends on how valuable knowing or not knowing what the differences would be here from the perspective of the docs reader.